packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -146,13 +147,19 @@ class DraggableScrollableSheet extends StatefulWidget {
   /// The default value is true.
   final bool expand;
 
+  /// A list of points that the widget should snap to. [minChildSize] and
+  /// [maxChildSize] are implicitly snapped to and should not be specified here-
+  /// pass an empty list to snap the widget between min and max.
+  final Set<double>? snapPoints;
```

Comment:
Made this a set because order isn't required and duplicate entries need not be supported. However, a list may be more intuitive for users. Should this remain a set or be made a list?